### PR TITLE
Follow-ups for a few foolish mistakes by me

### PR DESCRIPTION
**Pull request by YHNdnzj**

### CODE REVIEW

File: src/journal/journalctl.c [hunk 1]
```diff
@@ -330,6 +330,7 @@ static int parse_lines(const char *arg, bool graceful) {
 
 default_noarg:
         arg_lines = 10;
```
Comment (yuwata) [inline src/journal/journalctl.c:332]: Should be ARG_LINES_DEFAULT ?

**Review by yuwata — APPROVED** [review]
LGTM, one minor suggestion.

File: src/journal/journalctl.c (hunk 1 above)
Comment (YHNdnzj) [reply src/journal/journalctl.c:332]: No, this is applied when `-n` is specified without argument. `ARG_LINES_DEFAULT` is used to indicate `-n` is not specified at all, which is what we initialize `arg_lines` to.

Comment (yuwata) [reply src/journal/journalctl.c:332]: Ah, I see. Thanks. I thought `ARG_LINES_DEFAULT` is `10`, but it is not. Sorry for the noise.

**Review by bluca — APPROVED** [review]

**Pull request merged**